pymongo/encryption.py

```diff
@@ -296,7 +296,11 @@ def _get_internal_client(encrypter, mongo_client):
 
         io_callbacks = _EncryptionIO(metadata_client, key_vault_coll, mongocryptd_client, opts)
         self._auto_encrypter = AutoEncrypter(
-            io_callbacks, MongoCryptOptions(opts._kms_providers, schema_map)
+            io_callbacks,
+            MongoCryptOptions(opts._kms_providers, schema_map),
+            opts._csfle_path,
+            opts._csfle_required,
+            opts._bypass_auto_encryption,
```

Comment:
These are arguments for MongoCryptOptions, not AutoEncrypter.

Reply:
Done.